### PR TITLE
JavaScript2-Homework-Week-1

### PR DESCRIPTION
**Pull request by qamarfj**

### CODE REVIEW

File: javascript/javascript2/week1/shortestWord/shortest-word.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+"use strict"
+/**Write a function that finds the shortest word of an array of words */
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+
+function getShortestWord(danishWords) {
+    let shortestWord = danishWords[0];
+    for (const currentWord of danishWords) {
+        if (shortestWord.length > currentWord.length) {
```
Comment (Rassibassi) [inline javascript/javascript2/week1/shortestWord/shortest-word.js]: In the first loop iteration your code will always compare the first word with itself!
E.g., here in the first iteration both `currentWord` and `shortestWord` are "bil"
Which means there is always one unnecessary iteration. Can you change the for loop such that this is solved?

**Review by Rassibassi — COMMENTED** [review]
Good work, let me know if you have any questions regarding my comments

File: javascript/javascript2/week1/shortestWord/shortest-word.js (hunk 1 above)
Comment (qamarfj) [reply javascript/javascript2/week1/shortestWord/shortest-word.js]: Thank you for reviewing my homework. I have changed the loop

File: javascript/javascript2/week1/spirit/spirit.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ * fx Benjamin - The fullmoon wolf. */
+const animals = ['The lovely Bear', ' The beatifull Butterfly', 'The lazy Cat', 'The Greedy Deer', 'The Drowning Dolphin',
+    'The sleeping Lion', 'The flying Owl', 'The beatifull Peacock', 'The angry Tiger', 'The fullmoon wolf']
+const btnSpiritGenerator = document.getElementById("btn");
+
+
+btnSpiritGenerator.addEventListener("click", () => {
+    const userName = document.getElementById("userName").value;
+    if (userName.length > 0) {
+        const spiritName = `${userName}  -   ${animals[Math.floor(Math.random() * animals.length)]} .`;
```
Comment (soujanyapolavarapu) [inline javascript/javascript2/week1/spirit/spirit.js:13]: @qamarfj should we get random spirit animal with one name for every click ?

**Review by soujanyapolavarapu — COMMENTED** [review]
Hi Qamar, I have reviewed your home work, looks very good. Learned something from your code. I had a question and I  added comment please reply to that comment.Thankyou :) .

File: javascript/javascript2/week1/spirit/spirit.js (hunk 2 above)
Comment (qamarfj) [reply javascript/javascript2/week1/spirit/spirit.js:13]: No for every click we get new value from input field and new random spirit.

**Pull request merged**